WooCommerce/Classes/ViewRelated/Blaze/BlazeIntro/BlazeLearnHowView.swift

```diff
@@ -0,0 +1,242 @@
+    }
+
+    var stepBlock: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 0) {
+                VStack(alignment: .leading, spacing: 0) {
+                    ForEach(steps, id: \.title) { step in
+                        StepView(step: step)
+
+                        Divider()
+                            .frame(height: Layout.StepBlock.dividerHeight)
+                            .overlay(Color(uiColor: .systemBackground))
+                    }
+                }
+                .background(Layout.StepBlock.backgroundColor)
+            }
+            .cornerRadius(Layout.StepBlock.cornerRadius)
+        }
+        .padding(Layout.contentPadding)
+    }
+}
+
+private extension BlazeLearnHowView {
+    struct Step {
+        let number: String
+        let title: String
+        let subtile: String
+    }
+
+    struct StepView: View {
+        /// Scale of the view based on accessibility changes
+        @ScaledMetric private var scale: CGFloat = 1.0
+
+        let step: Step
+
+        var title: AttributedString {
+            var attributedText = AttributedString(.init(step.title + ":"))
+            attributedText.font = UIFont.font(forStyle: .subheadline, weight: .semibold)
+            attributedText.foregroundColor = .init(.text)
+            return attributedText
+        }
+
+         var subtitle: AttributedString {
+             var attributedText = AttributedString(.init(step.subtile))
+             attributedText.font = UIFont.font(forStyle: .subheadline, weight: .regular)
+             attributedText.foregroundColor = .init(.text)
+             return attributedText
+         }
+
+        var body: some View {
+            HStack(spacing: Layout.StepBlock.hSpacing) {
+                ZStack {
+                    Circle()
+                        .fill(Color(uiColor: .systemBackground))
+                        .frame(width: Layout.StepBlock.numberBackgroundSize * scale, height: Layout.StepBlock.numberBackgroundSize * scale)
+
+                    Text(step.number)
+                        .fontWeight(.semibold)
+                        .captionStyle()
+                }
+
+                Text(title + " " + subtitle)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+            }
+            .padding(Layout.StepBlock.padding)
+        }
+    }
+}
+
+private extension BlazeLearnHowView {
+    enum Localization {
+        static let title = NSLocalizedString(
+            "blazeLearnHowView.title",
+            value: "Learn how %1$@ works",
+            comment: "Title for the Blaze Learn How screen. %1$@ is replaced with string Blaze. Reads like: Learn how Blaze works"
+        )
+        enum ChooseProduct {
+            static let number = NSLocalizedString(
+                "blazeLearnHowView.chooseProduct.stepNumber",
+                value: "1",
+                comment: "Step number for Choose a product step"
+            )
```

Comment:
Please remove the localization for the numbers as they should stay fixed.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/11583/commits/5697ce1a49863666dbdac981540c3541b9e3b1ae